Automattic/newspack-plugin
# Update header and footer style

### All Submissions:






### Changes proposed in this Pull Request:

This PR updates the header and footer to look more like Gutenberg. Making it a little bit more compact. We also have a "Return to Dashboard" button (the N logo) which we didn't use to have.

__After__
![after](https://user-images.githubusercontent.com/177929/134881070-338352f5-5de4-47bd-b286-05b09029a6c1.png)

__Before__
![before](https://user-images.githubusercontent.com/177929/134881106-68692540-eac9-4c29-8750-12d445ab2b29.png)

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Check all the different wizards, including the set up.

### Other information:
